server/controllers/sessions-controller.js

```diff
@@ -1,19 +1,36 @@
 var passport = require('passport');
 
+function parseCSV(str) {
+  return str
+    .split(',')
+    .map(v => v.trim())
+    .filter(v => v);
+}
+
 exports.login = function(req, res, next) {
+  // Logout to support requesting additional scopes within a live session.
+  req.logout();
+
+  const additionalScopes = parseCSV(req.query.scope || '');
```

Comment:
Good point, Rag.

And added a TODO to use robot account credentials instead of users. I will fix that asap.